Source/Extensions/PDO/PDOException.cs

```diff
@@ -47,22 +47,25 @@ protected PDOException(SerializationInfo info, StreamingContext context)
         }
 
 #endif
-        #endregion
-
-        private PhpArray m_errorInfo;
-
-        private PDOException(ScriptContext context, PhpArray errorInfo)
-            : base(context, true)
-        {
-            this.m_errorInfo = errorInfo;
-        }
-
-        [PhpVisible]
-        public PhpArray errorInfo { get { return this.m_errorInfo; } }
-
-        public static void Throw(ScriptContext context, string message, PhpArray errorInfo, object code, object previous)
-        {
-            PHP.Library.SPL.Exception.ThrowSplException(ctx => new PDOException(ctx, errorInfo), context, message, code, previous);
-        }
-    }
-}
+        #endregion
+
+        private PhpArray m_errorInfo;
+
+        private PDOException(ScriptContext context, PhpArray errorInfo)
+            : base(context, true)
+        {
+            this.m_errorInfo = errorInfo;
+        }
+
+        [PhpVisible]
+        public PhpArray errorInfo { 
+            get { return this.m_errorInfo; }
+            set { this.m_errorInfo = value; }

```

Comment:
Sorry for the diff mess caused by GitHub online editor, this is the only changed line actually.